9c7718cddb229150791d8f8d13961979/update/sys_metadata_link_c36ac1a1c3ed5a10a8361dd1b40131db.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_metadata_link">
+    <sys_metadata_link action="INSERT_OR_UPDATE">
+        <directory>update</directory>
+        <documentkey>4ea905e1c36d5a10a8361dd1b40131ec</documentkey>
+        <payload>&lt;?xml version="1.0" encoding="UTF-8"?&gt;&lt;record_update table="x_snc_pointsthing_message"&gt;
+    &lt;x_snc_pointsthing_message action="INSERT_OR_UPDATE"&gt;
+        &lt;message&gt;&lt;![CDATA['Participating for the 1st time. This '+user+ ' has scored a' +points + ' points.']]&gt;&lt;/message&gt;
```

Comment:
This does not use correct grammar and does not have correct spacing. Additionally, the message about participation isn't suitable for this repo. This should be a celebration or kudos message, particularly for a milestone like 1000 points